integration_tests/packages.yml

```diff
@@ -1,7 +1,7 @@
 # packages:
 # - package: fivetran/apple_search_ads_source
 #   version: [">=0.1.0", "<0.2.0"]
-packages:
-  - git: https://github.com/fivetran/dbt_apple_search_ads_source.git
-    revision: main
-    warn-unpinned: false
\ No newline at end of file
+packages: ##
+  - git: https://github.com/fivetran/dbt_apple_search_ads_source.git ##
+    revision: main ##
+    warn-unpinned: false ##
```

Comment:
We can actually have this just be
```yml
packages:
  - local: ../
```
This way it installs the local package and doesn't need to be updated for future updates.